src/app/components/fileUploader.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Button } from "@mui/material";
+
+const MAX_FILE_SIZE = 2 * 1024 * 1024; // 2MB limit
```

Comment:
```suggestion
const MAX_FILE_SIZE = 30 * 1024 ; // 30KB limit
```
I feel like this is a little large, maybe we can make it smaller?